employee_class.py

```diff
@@ -10,109 +13,136 @@ def __init__(self, name, surname, age, employment_date, day_of_leaving, phone_nu
 
     def print(self):
         print(
-            f'Name: {self.name}| Surname: {self.surname}| Age: {self.age}| Employment date: {self.employment_date}|'
-            f'Date of leaving: {self.day_of_leaving}| Phone number: {self.phone_number}')
+            f'Name:\t{self.name}\tSurname:\t{self.surname}\tAge:\t{self.age}'
+            f'\tEmployment date:\t{self.employment_date}\t'
+            f'Date of leaving:\t{self.day_of_leaving}\t\tPhone number:\t{self.phone_number}')
 
     @staticmethod
     def selection():
-        selection = input('Enter your choice here (1, 2, 3, 4, 5 or 6): ')
-        if not selection:
-            print("Selection cannot be blank")
-        elif selection.isdigit() and 0 <= int(selection) < 7:
-            return int(selection)
-        else:
-            print("Please enter number between 1 and 6")
+        while True:
+            try:
+                selection = int(input('Enter your choice here (1, 2, 3, 4, 5 or 6): '))
+                if 0 <= int(selection) < 7:
+                    return selection
+                else:
+                    print("Invalid format. Please enter number between 1 and 6")
+            except ValueError:
+                print("Invalid format. Please enter number between 1 and 6")
 
     @staticmethod
     def selection_name():
         while True:
-            name = str(input("Name: "))
-            if not name:
-                print("Name cannot be blank")
-                continue
-            elif name.isalpha():
-                return name
-            else:
-                print("Name cannot contain numbers")
+            try:
+                x = Employee._validate_name(str(input("Name: ")))
```

Comment:
underscore metodo pradžioje reiškia, kad metodas yra private ir jis turi būti naudojamas tik klasės viduje (čia kiek bent jau skaičiau aš)

Reply:
aa, matau, kad čia ta pati klasė, tai teoriškai tu gali tuos metodus kviesti be ```Employee.```

Reply:
Underscore pašalinau. Galiu iškviesti be Employee metodus toje pačioje klasėje perdarydama metodus su self, tačiau tada negaliu jų iškviesti per employee menu.

![image](https://user-images.githubusercontent.com/94298574/145184640-8e9f9273-7df0-4241-997a-ac547df5b3f9.png)

![image](https://user-images.githubusercontent.com/94298574/145184761-be8f71a6-84fb-43af-8c4f-17428eebe20c.png)